### DISCUSSION TITLE
Mason commands are not working

### DISCUSSION THREAD

**Discussion by cheiktraore** [mason-org/mason.nvim#357] [Q&A] [answered]
Hi! I don't know why but I installed mason but no command is working. For instance I get this error message:
"E492: Not an editor command: Mason". This is how I installed it:
`  use({ "williamboman/mason.nvim" }) `
  `use({"williamboman/mason-lspconfig.nvim", after="mason.nvim"})` 
	`use({ "neovim/nvim-lspconfig", after="mason-lspconfig.nvim"}) -- enable LSP`

**Comment by williamboman** [comment, marked as answer]
Hello! Do you setup Mason anywhere? You'll need to call the `.setup()` functions for both `mason` and `mason-lspconfig` for the plugin to activate.

**Reply by LauGaHo** [reply]
I call the `.setup()` both `mason` and `mason-lspconfig`, but still not work.
```lua
   20 local mason_status, mason = pcall(require, "mason")
   19 if not mason_status then
   18   return
   17 end
   16
   15 local mason_lspconfig_status, mason_lspconfig = pcall(require, "mason_lspconfig")
   14 if not mason_lspconfig_status then
   13   return
   12 end
   11
   10 mason.setup()
    9
    8 mason_lspconfig.setup({
    7   ensure_installed = {
    6     "tsserver",
    5     "html",
    4     "cssls",
    3     "tailwindcss",
    2     "sumneko_lua",
    1   }
  21  })
```

**Comment by cheiktraore** [comment]
Thank you. It is the exact part I did not get.
